scripts/plotting/tem.py

```diff
@@ -189,8 +200,6 @@ def tem(adf):
         #Add plot to website (if enabled):
         adf.add_website_data(plot_name, "TEM", case_name, season=s)
 
-    print("  ...TEM plots have been generated successfully.")
```

Comment:
Is there a reason you got rid of this `print` statement?

Reply:
I probably accidentally over-rid this line, thanks